deploy/Dockerfile

```diff
@@ -7,18 +7,13 @@ LABEL operators.operatorframework.io.bundle.package.v1=ocs-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
 
-COPY bundle/manifests /manifests/
-COPY bundle/metadata/annotations.yaml /metadata/annotations.yaml
+COPY ocs-operator/manifests /manifests/
+COPY ocs-operator/metadata/annotations.yaml /metadata/annotations.yaml
 
 # These are three labels needed to control how the pipeline should handle this container image
 # This first label tells the pipeline that this is a bundle image and should be
 # delivered via an index image
 LABEL com.redhat.delivery.operator.bundle=true
```

Comment:
Do we need these labels anymore? They were here when we didn't have separate build methods for the bundle Downstream, But now the build team keeps their own separate build Dockerfiles, So I guess we can do away with these labels.

Reply:
If not anything else, these labels remind us of what is required for the release builds. Also, this PR is not a refactor, so I'd like to retain the existing content for consistency. We can always follow up with a refactor PR if needed.

Reply:
Hm, Okay. Will handle the cleanups later with refactor PRs